exception
===================================

These are custom exceptions for pyconll. Right now, this only consists of a ``ParseError``.


API
----------------------------------
.. automodule:: pyconll.exception
    :members:
    :special-members:
